### DISCUSSION TITLE
How can I add dynamic social banner?

### DISCUSSION THREAD

**Discussion by Tarbez** [timlrx/tailwind-nextjs-starter-blog#492] [Q&A]
Hello, I have been trying to add a social banner to my articles so that when I share them on social media, I will be able to see the picture as thumbnail like in twitter, linkedin, and so on.

```
title: 'Blog Title'
date: 2022-06-23T17:32:14Z
lastmod: '2022-06-23'
tags: ['tag-1, 'tag-2']
draft: false
summary: 'Blog quick description'
socialBanner: '/static/images/2022/june/3-CoversSet.png'
images: '/static/images/2022/june/3-CoversSet.png'
authors: ['default']
```

None of those worked. Not sure how I can do it?

**Comment by timlrx** [comment]
The `images` field should work. There's no `socialBanner`. If there's a problem, let me know the link to the post as well as to the repo. Thanks.

**Reply by Tarbez** [reply]
For the repo: [here](https://github.com/Tarbez/CR3TV-Blog-Starter), for the blog post [here](https://www.cr3tv.blog/blog/psychology/recovery/so-i-lost-all-my-work-from-the-last-7-years-now-what)

I tried posting on Linkedin the image did not show up.
When I go to [linkedin post inspector](https://www.linkedin.com/post-inspector/) I find these issues:

1. No image found
2. No author found
3. No publication date found


So I went to the SEO.js, inside the `<head> `and I wanted to add this line for the author but I think it is not going to work.
` <meta name="author" content={authors} />`
I would need to map through it and get the first one? 


However on Facebook it does show the thumbnail.

Twitter, discord, and probably more apps don't seem to read it.

Not sure how to fix this.

**Reply by nelsonlaidev** [reply]
Just change siteurl to your domain
https://github.com/Tarbez/CR3TV-Blog-Starter/blob/master/data/siteMetadata.js

**Reply by Tarbez** [reply]
Thank you! That worked for the image thumbnail, though the date and author are not found, any way to fix this?
